src/include/partitioning/custom_part_dialogs.rb

```diff
@@ -1674,8 +1674,7 @@ def SubvolHandling(old, new)
             )
             Popup.Message(tmp)
             pth = svtmp + pth
-          end
-          if Builtins.contains(SubvolNames(new), pth)
+          elsif Builtins.contains(SubvolNames(new), pth)
```

Comment:
Sorry, I just finished up my school exams today, I will take a look at the code now